YeRoBattle.Engine/BattleCalculator.cs

```diff
@@ -31,6 +32,62 @@ public void Hit(Character attacker, Character defender)
             {
                 defender.IsDead = true;
             }
+        }
+        public void Healing(Character character, Character target)
+        {
+            var heal = character.HealPower;
+
+            if (heal > 0) // heal character 
+            {
+                _logger.WriteLine($@"character {character.Name} has healed {character.Name} by {heal}");
+                character.Health = character.Health + heal;
```

Comment:
why do you change health property?

health - property that define MAX HP of character. 
We define it when we create a character

Reply:
fixed